src/Smoker/SmokeTest.cs

```diff
@@ -229,32 +286,21 @@ public async Task RunLoop(int id, Config config, CancellationToken ct)
                             using HttpResponseMessage resp = await _client.SendAsync(req).ConfigureAwait(false);
                             body = await resp.Content.ReadAsStringAsync().ConfigureAwait(false);
 
-                            int duration = (int)DateTime.UtcNow.Subtract(dt).TotalMilliseconds;
+                            double duration = Math.Round(DateTime.UtcNow.Subtract(dt).TotalMilliseconds, 0);
 
                             // validate the response
                             res = ValidateAll(r, resp, body);
 
+                            // check the performance
+                            var perfLog = GetPerfLog(r, string.IsNullOrEmpty(res), duration);
+
                             // only log 4XX and 5XX status codes
                             if (config.Verbose || (int)resp.StatusCode > 399 || !string.IsNullOrEmpty(res))
                             {
-                                if (config.RunWeb)
-                                {
-                                    // datetime is redundant for web app
-                                    Console.WriteLine($"{id}\t{(int)resp.StatusCode}\t{duration}\t{resp.Content.Headers.ContentLength}\t{r.Url}");
-                                }
```

Comment:
Does this check for config.RunWeb == true need to be added to the LogToConsole function?